plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/command/AssociationCreateCommand.java

```diff
@@ -137,6 +145,12 @@ public void undo() {
         if (association != null) {
             sourceEntity.removeAssociation(association, true);
             targetEntity.removeReferenceAssociation(association, true);
+
+            if (virtualFk != null) { 
+                virtualFk.getEntity().removeForeignKey(virtualFk);
+                virtualFk.getEntity().persistConfiguration();
```

Comment:
Configuration is already persisting in `ERDEditorEmbedded::doSave`. Not sure if it should be done here too.